tuya.cpp

```diff
@@ -466,9 +466,10 @@ void DeRestPluginPrivate::handleTuyaClusterIndication(const deCONZ::ApsDataIndic
             }
             else
             {
-                // Switch device 1/2/3 gangs
+                // Switch device 1/2/3 gangs or dimmer
                 switch (dp)
                 {
+                    // State
                     case 0x0101:
                     case 0x0102:
                     case 0x0103:
```

Comment:
Weren't there constants for the `dp` values?

Reply:
Heee ....
It the combinaison of dp type and dp value, 0x01 is the type and 0x01 0x02 0x03 tha value.
Both are defined in tuya.h file, and I m removing magic number for other file, but for the tuya one, I need to rewrite all the code.
I need to make 1 branch for type and inside 1 branch for value, so I can do it, but will be something sensible. But all the ugly part is located in the tuya file, can clean it myself, but need a good occasion.

Reply:
Ok thanks, that sounds good. It would be cool if that gets cleaned up in future PR's.
I really like for code where already some constants are used, this helps a lot during reviewing code :)

Reply:
Yeah I know, It s for that at every new PR I correct one of them in the rest of code, but not the same problem for the tuya.cpp file I have take a bad start.